lib/gettext-setup/pot.rb

```diff
@@ -71,12 +71,11 @@ def self.generate_new_pot(locales_path = GettextSetup.locales_path, path = nil)
 
     def self.generate_new_po(language, locales_path = GettextSetup.locales_path,
                              pot_file = nil, po_file = nil)
+      language ||= ENV['LANGUAGE']
```

Comment:
Oh, I see! I thought you wanted it to be included somehow in the initialize() method itself, and I was a little confused.